### PR TITLE
Add blog page

### PR DESCRIPTION
**Pull request by nastyapash**

### CODE REVIEW

File: qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{{ with (.Site.GetPage "/qdrant-blog/qdrant-blog-posts") }}
+  <section class="qdrant-blog-posts">
+    <div class="container">
+      <div class="qdrant-blog-posts__articles post">
```
Comment (trean) [inline qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html:4]: @nastyapash I suppose that you preferred custom styles over using bootstrap `.row > .col` classes to make gutters between posts wider as in the design 🙂 
But here is a way to do the same with Bootstrap (`.row .gx-5`) https://getbootstrap.com/docs/5.1/layout/gutters/

Could we please use this? 🙏

File: qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        {{ range $index, $element := .Params.posts }}
+          <article class="qdrant-blog-posts__article post">
```
Comment (trean) [inline qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html:6]: Does the `.post` class nest on purpose?

**Review by trean — CHANGES_REQUESTED** [review]
I think everything looks good, except for one minor change that I would like to request.

File: qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html (hunk 1 above)
Comment (nastyapash) [reply qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html]: done!

File: qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html (hunk 2 above)
Comment (nastyapash) [reply qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html]: yes, that's unnecessary

**Pull request merged**